packages/shared/src/components/squads/SharePostBar.tsx

```diff
@@ -73,7 +73,11 @@ function SharePostBar({
         )}
       >
         <LockIcon size={IconSize.Small} />
-        <p className="typo-callout">Only admins and moderators can post</p>
+        <p className="typo-callout">
```

Comment:
This is the disabled render only right, not sure why this change was needed then?

Reply:
This copy is from open squad page design.

Reply:
But do they not intertwine?
Just making sure the evaluation is ok because the setting to even render this might be different.